app/views/state_file/questions/federal_dependents/index.html.erb

```diff
@@ -3,39 +3,31 @@
 
 <% content_for :body_class, "dependents-index" %>
 
```

Comment:
this looks like a lot of changes but i'm just replacing `content_for :main` with `content_for :card` and removing the redundant wrapping elements